### ISSUE TITLE
Big black circle cover original dots when using "hover" with linechart

### ISSUE THREAD

**Issue by superbull** [DmitryBaranovskiy/g.raphael#166]
For example:
var paper = Raphael('canvas');
paper.linechart(
                10, 10, 300, 300, 
                [[3.5, 4.5, 5.5, 6.5, 7, 8]], [ [10, 20, 30, 25, 15, 28]],
                { 
                    nostroke: false, 
                    axis: "0 0 1 1", 
                    symbol: "circle", 
                    smooth: true,
                }
            )
            .hover(function () {}, function(){});

I guess on "g.raphael / g.line.js line#272", the "opacity: 1 " should be changed to "opacity: 0"?

**Comment by noahp** [comment]
Agree- should match what is done for createColumns- {stroke:"none",fill:"#000",opacity:0}.
